app/src/main/java/xyz/jhughes/laundry/activities/LocationActivity.java

```diff
@@ -37,18 +40,27 @@
     @Bind(R.id.location_activity_toolbar) Toolbar toolbar;
     @Bind(R.id.progressBar) ProgressBar mLoadingProgressBar;
 
-
     private LocationAdapter adapter;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+
+        if (!getIntent().getBooleanExtra("forceMainMenu", false)) {
+            String lastRoom = SharedPrefsHelper.getSharedPrefs(this)
+                    .getString("lastRoom", "hi, jeff");
+            if (!lastRoom.equals("hi, jeff")) {
+                Intent intent = new Intent(this, MachineActivity.class);
+                intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
```

Comment:
Ah, that was from a different attempt that didn't work. I forgot to take it away. I'll do that